src/components/AssetManagementPanel/AssetManagementForm/FormVariants/AcceptSurrenderedForm/AcceptSurrenderedForm.tsx

```diff
@@ -79,15 +84,15 @@ export const AcceptSurrenderedForm: FunctionComponent<AcceptSurrenderedFormProps
               </div>
               <div className="w-auto ml-2">
                 <Button
-                  className="bg-red-500 text-white hover:bg-red-700"
+                  className="bg-rose text-white rounded-xl text-lg py-2 px-3 shadow-none hover:bg-rose-400"
                   onClick={handleDestroyToken}
                   disabled={isDestroyTokenPendingConfirmation}
                   data-testid={"acceptSurrenderBtn"}
                 >
                   {isDestroyTokenPendingConfirmation ? (
                     <LoaderSpinner data-testid={"accept-loader"} />
                   ) : (
-                    <>Shred Document</>
+                    <>Accept Document</>
```

Comment:
text wise, no longer shred?

Reply:
reverted back to Shred Document after discussion.